app/src/pages/Play/index.tsx

```diff
@@ -45,69 +41,30 @@ function Play() {
     (player) => player.id === activeTurn?.player_id
   )
 
-  const [activeTurnPlayState, setActiveTurnPlayState] = React.useState(
-    ActiveTurnPlayState.Waiting
-  )
-
-  const startingSeconds =
-    activeTurn?.seconds_per_turn_override || currentGame.seconds_per_turn || 0
-
-  const [secondsLeft, setSecondsLeft] = React.useState(
-    activeTurn?.started_at
-      ? calculateSecondsLeft(
-          dateFromTimestamptzNow(activeTurn.started_at),
-          startingSeconds
-        )
-      : startingSeconds
-  )
-
-  // change in settings
-  React.useEffect(() => {
-    if (currentGame.seconds_per_turn) {
-      setSecondsLeft(currentGame.seconds_per_turn)
+  const getActiveTurnPlayState = () => {
+    if (activeTurn?.review_started_at) {
+      return ActiveTurnPlayState.Reviewing
```

Comment:
we could refactor as 

```
const getPlayStateFromTurn = (turn: TurnFragment) => {
  if (turn.review_started_at) {
    return ActiveTurnPlayState.Reviewing
  }

  if (turn.started_at) {
    return ActiveTurnPlayState.Playing
  }

  return ActiveTurnPlayState.Waiting
}
```
and remove this function from the component entirely, making deps simpler